src/Sulu/Bundle/AdminBundle/Resources/js/components/Hint/Hint.js

```diff
@@ -0,0 +1,24 @@
+// @flow
+import React from 'react';
+import Icon from '../Icon';
+import hintStyles from './hint.scss';
+
+type Props = {||};
+
+export default class Hint extends React.Component<Props> {
```

Comment:
Not sure about the name `Hint` should we go with `ErrorHint`?

Reply:
`ErrorHint` says implicitly that this should be used when an error happens, but this is also used for the "no permission" message, which is technically not an error

therefore I think I would go with `Hint` or maybe `Indication`/ `Notice` ? Tbh I cannot really decide what would be the most suitable naming for this component 😅

Reply:
Would stay then with `Hint`